schemas/Clinical/index.js

```diff
@@ -415,7 +415,7 @@ const resolvers = {
       const { shortName } = args;
 
       const response = await clinicalService.getRegistrationData(shortName, Authorization);
-      return convertRegistrationDataToGql({ ...response, shortName });
+      return convertRegistrationDataToGql({ registration: response, shortName });
```

Comment:
is the `shortName` supposed to go under registration as well?

Reply:
it is their inside registration from clinical, but it looks like shortName is used by `convertRegistrationDataToGql`, I tried to keep my changes to a minimum :slightly_smiling_face:

Reply:
in case of no registration, the UI still needs the `shortName` there because it's used as the cache key, so I think that's why we include the `shortName` from `args`